### PR TITLE
clarifying the readmes

### PR DESCRIPTION
**Pull request by PatrickAlphaC**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,27 +4,55 @@ This repository contains all relevant project necessary to have [polkadot](https
...
+You can see in the logs of `./setup` if the address has moved.
+
+_note_
+If you'd like to restart you will have to remove all the docker containers. You can do this by running:
 
-You are now ready to send test requests and see the result being provided back by an Oracle.
+```
+docker kill substrate-chainlink_external-initiator-node3_1 substrate-chainlink_external-initiator-node2_1 substrate-chainlink_external-initiator-node1_1 substrate-chainlink_chainlink-node1_1 substrate-chainlink_chainlink-node3_1 substrate-chainlink_chainlink-node2_1 substrate-chainlink_postgres-node1_1 substrate-chainlink_substrate-adapter1_1 substrate-chainlink_substrate-adapter3_1 substrate-chainlink_postgres-node2_1 substrate-chainlink_postgres-node3_1 substrate-chainlink_substrate-adapter2_1
+
+docker system prune
```
Comment (boxhock) [inline README.md:50]:
```suggestion
```

Should not be necessary

File: README.md [hunk 2]
```diff
@@ -4,27 +4,55 @@ This repository contains all relevant project necessary to have [polkadot](https
...
+
+docker volume rm substrate-chainlink_cl1 substrate-chainlink_cl2 substrate-chainlink_cl3 substrate-chainlink_pg1 substrate-chainlink_pg2 substrate-chainlink_pg3
```
Comment (boxhock) [inline README.md:52]:
```suggestion
docker-compose rm
```

Untested, but should work.

File: README.md [hunk 3]
```diff
@@ -4,27 +4,55 @@ This repository contains all relevant project necessary to have [polkadot](https
...
+To spin up some simple Chainlink nodes to intereact with the parachain you just created, run the setupcommand:
+`cd substrate-chainlink`
+`./setup`
+
+This will run a number of docker commands that will spin up your Chainlink node. You can access them by going to your browser at:
+`http://localhost:6691/`
+You can see in the logs of `./setup` if the address has moved.
+
+_note_
+If you'd like to restart you will have to remove all the docker containers. You can do this by running:
```
Comment (boxhock) [inline README.md:45]: What do you mean by restart?

You should be able to do `docker-compose down` to stop everything and `docker-compose up` to get back to where you were.

To reset everything, you need to run what's below.

Just to clarify, the `./setup` script is a one-time initialization script. You only have to run the docker containers to run the demo.

File: README.md (hunk 2 above)
Comment (PatrickAlphaC) [reply README.md:52]: Just tested, you need to use the "volume rm" command (or at least the `docker-compose down` command did not work)

File: README.md (hunk 3 above)
Comment (PatrickAlphaC) [reply README.md:45]: Ah ok! This is good information. I'll update the PR. thanks for your edits!

**Pull request merged**